### ISSUE TITLE
question about Train

### ISSUE THREAD

**Issue by lzjjia** [asrafulashiq/dynamic-cdfsl#11]
In your paper -"In the first step, we train our network only on the miniImageNet dataset for 200 epochs. We use SGD with momentum 0.9, weight decay 1e-4, learning rate 0.01, batch size 32, and the cosine learning rate scheduler".I would like to ask if I can increase the batch_size in the first step.
